sql/synth_recipes.sql

```diff
@@ -1323,7 +1323,7 @@ INSERT INTO `synth_recipes` VALUES (19920,1,0,0,23,0,0,0,0,0,0,4100,4242,16704,0
 -- INSERT INTO `synth_recipes` VALUES (19926,1,0,0,255,0,0,0,0,0,0,4100,4242,12816,0,0,0,0,0,0,0,649,817,818,850,2,6,1,2,'Scale Cuisses (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (19927,1,0,0,255,0,0,0,0,0,0,4100,4242,16466,0,0,0,0,0,0,0,707,651,651,651,1,1,1,1,'Knife (desynth)');
 -- INSERT INTO `synth_recipes` VALUES (19928,1,0,0,255,0,0,0,0,0,0,4100,4242,16631,0,0,0,0,0,0,0,895,801,651,651,1,1,2,2,'Kaiser Sword (desynth)');
--- INSERT INTO `synth_recipes` VALUES (19929,1,0,0,255,0,0,0,0,0,0,4100,4242,16475,0,0,0,0,0,0,0,653,653,653,653,1,1,1,1,'Mythril Kukri (desynth)');
+INSERT INTO `synth_recipes` VALUES (19929,1,0,0,42,0,0,0,0,0,0,4100,4242,16475,0,0,0,0,0,0,0,853,716,653,653,1,1,1,1,'Mythril Kukri (desynth)'); -- on synth level, used XI results
```

Comment:
comment unclear. "on synth level"? meaning desynth lv based on the synth lv? "used XI results", meaning ffxiclopedia?

Reply:
Sorry yeah... was trying to keep the comments shorter.

Reply:
expand them out just a little bit so we can tell the meaning at a glance. XI could be easily mistaken for other things. "used synth lv" would suffice and is only 2 more characters for that part